cloudhub-2/modules/ROOT/pages/ch2-configure-horizontal-autoscaling.adoc

```diff
@@ -0,0 +1,60 @@
+= Configuring Horizontal Autoscaling (HPA) for CH2 Deployments
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can configure horizontal scaling of Mule applications to make them responsive to resource usage by automatically scaling up or down the deployment replicas as needed.
+
+In Kubernetes, a Horizontal Pod Autoscaler (HPA) automatically updates a workload resource, with the aim of automatically scaling the workload to match demand. Horizontal scaling means that the response to increased load is to deploy more pods. For more information, visit the https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale[Kubernetes documentaton^]. 
+
+== Before You Begin
```

Comment:
CH 2.0 is a fully managed offering, this is not applicable/required for CH 2.0. Only RTF customers need to do this